downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,6 +60,20 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
+- Introduced the `Extract to transform function` code action to extract a value expression into a transform function.
+- Introduced the `Surround with lock` code action to wrap an isolated variable access with a `lock` statement.
```

Comment:
Think out loud, wouldn't "Wrap with lock" have been better than "Surround"? You're description uses wrap too.

Reply:
You're correct. However, renaming the code action would introduce inconsistency with existing ones, like `Surround with do/on-fail`.

Reply:
Ack. Let's keep as is.